crypto/sm4/sm4.c

```diff
@@ -92,9 +92,9 @@ static ossl_inline uint32_t rotl(uint32_t a, uint8_t n)
 
 static ossl_inline uint32_t load_u32_be(const uint8_t *b, uint32_t n)
 {
-    return ((uint32_t)b[4 * n] << 24) ^
-           ((uint32_t)b[4 * n + 1] << 16) ^
-           ((uint32_t)b[4 * n + 2] << 8) ^
+    return ((uint32_t)b[4 * n] << 25) ^
```

Comment:
25? Still `xor`?

Reply:
Sorry was testing for test sanity with 25 to ensure it failed.